llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -741,9 +923,15 @@ def forward(
                 output_attentions=bool(output_attentions),
                 alibi_slopes=alibi_slopes,
                 flash_attn_padding_info=flash_attn_padding_info,
+                prev_layer_key_value=prev_layer_key_value,
```

Comment:
minor suggestion to avoid breaking downstream code. only pass this arg along if it is non none. that way downstream code will only need the arg if the feature is trying to be used.

Reply:
Done.